external/appplatform/api/vmw_v1alpha1/supervisorprops_types.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2024 Broadcom. All Rights Reserved.
+// Broadcom Confidential. The term "Broadcom" refers to Broadcom Inc.
+// and/or its subsidiaries.
+
+package vmw_v1alpha1
```

Comment:
would prefer the package structure to look like 
external/(appplatform|supervisorservices)/api/v1alpha1. Aliases can be `appv1a1` or something similar instead of vmw_v1alpha1 everywhere

Reply:
Done.